content/Teams/sendwaste/index.md

```
@@ -0,0 +1,120 @@
+## 👥 Team Members
+
+- **Bethel Clement, MSc.** — Product Manager & Founder
+- **Oluokun Adesina** — Full Stack & Blockchain Developer
+- **Bridget** — UI/UX Designer
+- **Timothy Jehwo** — Backend Developer
+- **Daniel Adewole Adediran** — Data Scientist
+
+---
+
+## Pages
+## Pages
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)